### PR TITLE
chore(masthead ): applied tokens

### LINKED ISSUE #5712: Consume tokens in: Masthead and Navigation
[Figma design for Masthead](https://www.figma.com/file/VMEX8Xg2nzhBX8rfBx53jp/Penta-Component-Library?type=design&node-id=3%3A33&mode=design&t=doZDRUaxE4vsn5dD-1)

[Figma design for Navigation](https://www.figma.com/file/VMEX8Xg2nzhBX8rfBx53jp/Penta-Component-Library?type=design&node-id=3%3A35&mode=design&t=doZDRUaxE4vsn5dD-1)
```[tasklist]
### Tasks
- [ ] https://github.com/patternfly/patternfly/issues/6000
- [ ] https://github.com/patternfly/patternfly/issues/5999
```

### LINKED ISSUE #5999: Consume tokens in: Masthead

### PR DESCRIPTION
**Pull request by mattnolting**
closes #5999

Went ahead and update `toolbar` vars -> root as it's used frequently with `masthead`.

### CODE REVIEW

**Review by srambach — COMMENTED** [review]
I'd love a run through to process the hbs changes.

File: src/patternfly/components/Masthead/masthead-logo.hbs [hunk 1]
```diff
@@ -1,6 +1,29 @@
-<div class="{{pfv}}masthead__logo{{#if masthead-logo--modifier}} {{masthead-logo--modifier}}{{/if}}"
-  {{#if masthead-logo--attribute}}
-    {{{masthead-logo--attribute}}}
-  {{/if}}>
-  {{> @partial-block}}
-</div>
+<?xml version="1.0" encoding="UTF-8"?>
+<svg height="36px" viewBox="0 0 679 158" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (mcoker) [inline src/patternfly/components/Masthead/masthead-logo.hbs:2]:
```suggestion
<svg height="36px" viewBox="0 0 679 158" role="img">
```

File: src/patternfly/components/Masthead/masthead-logo.hbs [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+  <title>PF-HorizontalLogo-Color </title>
```
Comment (mcoker) [inline src/patternfly/components/Masthead/masthead-logo.hbs:3]: This should probably be similar to whatever the `alt` attribute would be 

```suggestion
  <title>Patternfly logo</title>
```

**Review by mcoker — CHANGES_REQUESTED** [review]
Just a couple o' small changes, and it's worth noting that the designs have a bit different padding, maybe something we can look at later.

**Review by mcoker — APPROVED** [review]

**Pull request merged**